modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -704,7 +704,7 @@ export class UtxoPsbt<Tx extends UtxoTransaction<bigint> = UtxoTransaction<bigin
    */
   getSignatureValidationArray(
     inputIndex: number,
-    { rootNodes }: { rootNodes?: Triple<BIP32Interface> } = {}
+    { rootNodes }: { rootNodes: Triple<BIP32Interface> }
```

Comment:
and mention the behavior/footgun in jsdoc
```suggestion
    { rootNodes }: { rootNodes: Triple<BIP32Interface> | undefined }
```